.env

```diff
@@ -1,3 +1,4 @@
 QA_DOMAIN=https://qa-libraryregistry.librarysimplified.org/admin
```

Comment:
We should rename this to `REGISTRY_ADMIN_DOMAIN` since it will change based on the environment, qa or production. Then we should move `LOGIN`, `REFRESH`, and `FETCH_LIBRARIES` out into a `constants.ts` file. This is because I want to limit the number of env vars I have to define in the Dockerfile. Anything in the `.env` can be values that change depending on environment.